src/multi.rs

```diff
@@ -593,6 +607,74 @@ impl Multi {
         }
     }
 
+    fn _timeout_i32(timeout: Duration) -> i32 {
```

Comment:
It's ok to not prefix this method with an underscore since it's already private anyway.